### DISCUSSION TITLE
Embed video in README.md

### DISCUSSION THREAD

**Discussion by danpoynor** [community/community#4818] [Other Feature Feedback, Questions, & Ideas] [answered]
In my README.md I'd like to include a brief video demonstrating the UX I've coded.

Currently I can paste a link to a Vimeo screen recording in the markdown, but when folks click it they leave the repo area on github.com and are taken to vimeo.com. I understand it's not possible to target a new browser tab/window with a link in markdown.

Ultimately it would be great if folks could view a video embedded within the readme while on the same page.

I've tried uploading a .mp4 file directly based on the new [video uploads feature](https://github.blog/2021-05-13-video-uploads-available-github/), but currently it has a 10 MB file size limit which is small for a demo.

A way to embed Vimeo or other video providers would be great.

**Comment by danpoynor** [comment, marked as answer]
I noticed Pro account customers have a 100 MB limit which will work great!

**Comment by scunliffe** [comment]
Not quite the solution you're after, but if you want to put a link in that opens in a new tab... you're stuck from the markdown side, but basic HTML should still work:

`<a href="https://vimeo.com/127761531" target="_blank">Watch video!</a>`

Renders as: <a href="https://vimeo.com/127761531" target="_blank">Watch video!</a>
(ugh... this doesn't work anymore... the `target` attribute gets stripped out... *much sad*)

**Comment by Orangee92** [comment]
I created a portfolio website on VSC and added it to GitHub desktop. All my mp4 videos won't show up, how to make it work?
